pkg/install/gatherlogs.go

```diff
@@ -0,0 +1,69 @@
+package install
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	consoleapi "github.com/openshift/console-operator/pkg/api"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// GatherFailureLogs will log failed cluster components data out into RP logs for
+// easier debugging
+func (i *Installer) GatherFailureLogs(ctx context.Context) error {
+	err := i.initializeKubernetesClients(ctx)
+	if err != nil {
+		return err
+	}
+
+	var statusCode int
+	err = i.kubernetescli.Discovery().RESTClient().
+		Get().
+		AbsPath("/healthz").
+		Do().
+		StatusCode(&statusCode).
+		Error()
+
+	if statusCode != http.StatusOK {
+		return fmt.Errorf("cluster never reached healthy API server state")
+	}
+
+	err = i.logClusterOperatorStatus(ctx)
+	if err != nil {
+		return err
+	}
+	return i.logConsoleOperatorStatus(ctx)
```

Comment:
the issue we're seeing isn't with the console operator, it is just that we happen to be waiting for the console operator.  First and foremost need to log the cluster version CR.